default.nix

```diff
@@ -10,21 +10,35 @@ let
   # If haskellNix is not found run:
   #   niv add input-output-hk/haskell.nix -n haskellNix
 
+  freeglut-overlay = self: super:
+    {
+      freeglut = super.freeglut.overrideAttrs(old: if !self.stdenv.hostPlatform.isWindows then {} else {
+        buildInputs = [ ];
```

Comment:
`freeglut` in `nixpkgs` has a bunch of `buildInputs` that are Linux-specific:

```
  buildInputs = [ libXi libXrandr libXxf86vm libGL libGLU xlibsWrapper ];
```

We actually don't need anything here since mingw provides OpenGL for us.